demo2/db/migrations/20210513225012-create-products.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('products', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      product_name: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      price: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      ingridients: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      amount: {
+        type: Sequelize.INTEGER,
+        allowNull: false
+      },
+      manufacture_id: {
+        type: Sequelize.INTEGER,
+        allowNull: false
+      },
+      category_id: {
+        type: Sequelize.INTEGER,
+        allowNull: false
+      },
+      unit_id: {
+        type: Sequelize.INTEGER,
+        allowNull: false
+      },
+      img_link: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
```

Comment:
why everything is with allowNull: false ? there could be situation when for example link is not presented